src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -159,11 +159,6 @@ public void RefreshWorkloadManifests()
                 return _workloadSet?.Version!;
             }
 
-            if (InstallStateContents.FromPath(Path.Combine(WorkloadInstallType.GetInstallStateFolder(_sdkVersionBand, _sdkRootPath), "default.json")).UseWorkloadSets == true)
```

Comment:
We'd want this to change to echo the version for the current SDK, per our discussion this morning.

Reply:
I've filed an issue to track the further changes needed: https://github.com/dotnet/sdk/issues/40005